pkg/generate/app/sourcelookup.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var (
-	argumentGit         = regexp.MustCompile("^(http://|https://|git@|git://).*\\.git(?:#([a-zA-Z0-9]*))?$")
+	argumentGit         = regexp.MustCompile("^(http://|https://|git@|git://).*(?:#([a-zA-Z0-9]*))?$")
```

Comment:
Can we cross that bridge when we get there? There's likely some other way we can identify that other object depending on what it is.